packages/block-library/src/social-links/edit.js

```diff
@@ -131,51 +130,48 @@ export function SocialLinksEdit( props ) {
 						isAlternate: true,
 					} }
 				/>
-				<ToolbarGroup>
```

Comment:
As far as I understand, `BlockControlSlot` already returns a `ToolbarGroup` and should not be necessary anymore.

Not sure if the `Size` dropdown should live in the `Other` section instead of the block one. It seems to me it belongs in `Block` as it's a formatting option, but other options that don't have an icon but a text are usually in `Other` (as in the Image block)

Reply:
I have a small preference for it in "Other", to be honest. While it's technically a block level change, it feels different enough in how it works that it deserves its own box: https://developer.wordpress.org/block-editor/how-to-guides/designers/block-design/#group-block-toolbar-controls-with-related-items

Reply:
Doesn't it work in a _similar_ way to the heading level, heading markup aside? 🤔

Reply:
Well, yes and no. It add a size CSS class. I wouldn't object to it landing where you put it — the overall reduction of borders is worth it. But I think that in addition to the general hierarchy principles, "Other" also exists to check that box of "feels right". In this case, every instance of text-only buttons we have in the toolbar sit in their own groups with borders on the side. I think that's something to follow up on, in the design side, but to simply match the currrent structure with Replace buttons on images, I'd still keep "Size" in its own "Other" group. Not a very strong opinion, and okay to merge if this needs to land soon.

Reply:
The "text-only" factor is in my opinion reason enough to put it in a separate group, it doesn't _feel_ right design-wise to mix it with the other buttons.